src/main/java/me/thegiggitybyte/dsharpbridge/DSharpBridge.java

```diff
@@ -129,10 +130,22 @@ private void proxyMessageToMinecraft(MessageCreateEvent event) {
         
         if (author.getRoleColor().isPresent())
             userStyle.withColor(author.getRoleColor().get().getRGB());
+
+        var replyText = new LiteralText("");
+        var optionalMessageReference = event.getMessage().getMessageReference();
+        if(optionalMessageReference.isPresent() && optionalMessageReference.get().getMessage().isPresent()) {
+            var replyMessage = optionalMessageReference.get().getMessage().get();
+            var replyAuthor = replyMessage.getAuthor();
+            var replyInfoText = replyAuthor.getDiscriminatedName() + " (" + replyAuthor.getIdAsString() + ")\n\n" + replyMessage.getReadableContent();
+            var replyHoverEvent = HoverEvent.Action.SHOW_TEXT.buildHoverEvent(Text.of(replyInfoText));
```

Comment:
`Text.of()` is a client-side method which will cause a crash on the dedicated server; could you replace that with `new LiteralText()` please?